mip_convert/mip_convert/configuration/cv_config.py

```diff
@@ -162,11 +162,15 @@ def parent_activity_id(self, experiment_id, parent_experiment_id, mip_era):
             return self._UNKNOWN
         # Otherwise attempt to get parent activity id from parentage
         # dictionary
-        parentage_dict = self._get_parentage_dictionary(info)
-        if parent_experiment_id not in parentage_dict:
-            error_message = 'Parent experiment id "{}" not found in parentage dictionary: "{}"'
-            raise RuntimeError(error_message.format(parent_experiment_id, parentage_dict))
-        return parentage_dict[parent_experiment_id]
+        if mip_era == 'CMIP6':
```

Comment:
Took me a moment to work out what is going on here. Would adding this comment be useful?
```suggestion
        # CMIP6 had multiple options for the parent activity id of an experiment, which needed to be
        # handled separately. Not sure if this is true for CMIP7 yet.
        if mip_era == 'CMIP6':
```